### DISCUSSION TITLE
Battery Health Custom Capacity & Range

### DISCUSSION THREAD

**Discussion by nyccowgirl** [jheredianet/Teslamate-CustomGrafanaDashboards#92] [Q&A]
I apologize if this has been asked but I couldn't seem to find the instructions. I purchased my Tesla used so I am trying to customize the capacity and range to when it was new. However, it seems to default to zero each time I go to the dashboard. I tried to save the json file and override the battery-health.json in the teslamate/grafana/dashboards folder but it still shows as 0 (although I can see the changed json file). I have installed the extended dashboards hoping that it included something for us to change but since it's been integrated as a regular dashboard, I think it pulls the same default of 0. Is there a way for me to save the 72.6 and 259 (I have a 2016 MS 75D) without having to manually input it each time to calculate the degradation? TIA

**Comment by jheredianet** [comment]
Since it's Grafana, there is no way to save permanent values, unless you clone it in other dashboard for your own use. 
But there is a workaround for that: Open the dashboard, put your custom values and then save it as a bookmark on your Browser. So next time instead of opening that dashboard from TeslaMate, open it from your bookmarks.

**Comment by nyccowgirl** [comment]
Thank you @jheredianet !
